students/Isabella_Kemp/lesson03/list_lab.py

```diff
@@ -0,0 +1,83 @@
+# Isabella Kemp
+# 1/19/2020
+# list lab
+
+# Series 1
+# Create a list that displays Apples, Pears, Oranges, Peaches and display the list.
+List = ["Apples", "Pears", "Oranges", "Peaches"]
```

Comment:
Variables should be lowercase, and you want to avoid reserved words such as this.